### PR TITLE
Add/Update CDC Vaccination Age Groups

### LINKED ISSUE #1750: Add the 0-5 age demographic for COVID vaccination national
**Is your feature request related to a problem? Please describe.**
Currently, when looking at national vaccinations by age, the tracker only shows ages 5-12 as the youngest group. 

As the CDC mentions in [this newsletter](https://www.cdc.gov/coronavirus/2019-ncov/covid-data/covidview/index.html?ACSTrackingID=USCDC_2145-DM87783&ACSTrackingLabel=8.12.2022%20-%20COVID-19%20Data%20Tracker%20Weekly%20Review&deliveryName=USCDC_2145-DM87783) they now provide vaccination numbers for ages `0-2` and `2-4`. However it seems the [API we use only](https://data.cdc.gov/Vaccinations/COVID-19-Vaccination-Demographics-in-the-United-St/km4m-vcsb) provides a `<5` age group.

**Describe the solution you'd like**
We should add first dose vaccination numbers for the `<5` age groups in the national geo level.

### PR DESCRIPTION
**Pull request by eriwarr**
## Description

This PR updates the age groups of people with at least one dose administered as provided by the CDC. 

Age groups added: `<2` & `2-4`
Age groups updated: `25-39` to `25-49`; `65-74` & `75+` to `65+`

## Motivation and Context

Closes #1750

## How has this been tested?

Pytests and on Google Cloud

## Screens
Screen Shot 2022-08-25 at 10 50 28 AM
hots (if appropriate):

## Types of changes

- New feature (non-breaking change which adds functionality or content)

### CODE REVIEW

File: python/datasources/cdc_vaccination_national.py [hunk 1]
```diff
@@ -28,14 +28,14 @@
 }
 
 CDC_AGE_GROUPS_TO_STANDARD = {
+    'Ages_<2yrs': '<2',
```
Comment (benhammondmusic) [inline python/datasources/cdc_vaccination_national.py:31]: Did we talk about options for under 2? In incarceration we use `0-17` for under 18, but I'm open to other ideas like `Under 2` or `<2` we should discuss what's the easiest to understand

Comment (eriwarr) [reply python/datasources/cdc_vaccination_national.py:31]: Great point!

Comment (eriwarr) [reply python/datasources/cdc_vaccination_national.py:31]: Maybe we just do the `0-2` in order to keep it consistent with the rest of the site. What do you think @joshzarrabi?

Comment (joshzarrabi) [reply python/datasources/cdc_vaccination_national.py:31]: Yeah lets do `0-2`

**Comment by joshzarrabi** [comment]
@eriwarr Nice, dd you run the pipeline on your GCP account?

**Review by joshzarrabi — APPROVED** [review]

**Pull request merged**